frontend/public/components/terminal.jsx

```diff
@@ -33,6 +37,17 @@ export class Terminal extends React.Component {
     this.terminal && this.terminal.focus();
   }
 
+  setFullscreen_( fullscreen ) {
+    const masthead = document.querySelector('[role="banner"]');
+    const sidebar = document.getElementById('sidebar');
+    masthead.style.visibility = sidebar.style.visibility = fullscreen ? 'hidden' : 'visible';
```

Comment:
Just wondering if it works to give the terminal a higher z-index instead of trying to hide the masthead and sidebar. I'm worried this is going to keep breaking.

Reply:
Hi, ok switched to useing a `z-index` higher than the term fullscreen and masthead & sidebar.

Reply:
..this did resolve the hamburger menu showing through.